uint8 detectionId
float64[3] position
float64 confidence
float64 timestamp
